src/command.ts

```diff
@@ -29,104 +36,116 @@ export type CypressConfigEnv = {
   visualRegressionFailSilently?: boolean
 }
 
-export type TakeScreenshotProps = {
-  path: string
-  size: number
-  dimensions: {
-    width: number
-    height: number
-  }
-  multipart: boolean
-  pixelRatio: number
-  takenAt: string
-  name: string
-  blackout: string[]
-  duration: number
-  testAttemptIndex: number
-}
+type TakeScreenshotProps = Partial<Cypress.ScreenshotOptions> & { path: string }
```

Comment:
Do you want to make it partial? that means that "all" the props defined in Cypress.ScreenshotOptions will be optional

Also looking at the definition of [Cypress.ScreenshotOptions ](https://github.com/cypress-io/cypress/blob/ebbf7b1bdca49e509d22738c7b3ba92b26b065f3/cli/types/cypress.d.ts#L3669)

It looks like you want to use the types defined under `onAfterScreenshot` which is a function that accepts as a second argument the object that you are typing, but I don't know if there is an easy way to type it like that.

Because you can use a type like  `Cypress.ScreenshotOptions['onAfterScreenshot']` but that will return a function and what you need to type is the second parameter passing to that function, so you can either try to find aTS way to type that or just keep the old typeDefinitiopn and add the path

Reply:
Thanks for pushing this, it annoyed me as well. In the end, I saw that we don't need a type at all, seems that Cypress fixed their type:
<img width="371" alt="image" src="https://github.com/user-attachments/assets/ef1be4a0-49f7-4e17-ae90-0252b5c07b10">